File: 1293-three-consecutive-odds/1293-three-consecutive-odds.kt
class Solution {
    fun threeConsecutiveOdds(arr: IntArray): Boolean {
        var cnt = 0
        for (n in arr) {
            cnt = if (n % 2 == 1) cnt + 1 else 0
            if (cnt == 3) return true
        }
        return false
    }
}

File: 1293-three-consecutive-odds/1293-three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int cnt = 0;
        for (int num : arr) {
            if (num & 1) {
                cnt++;
                if (cnt == 3) return true;
            } else cnt = 0;
        }
        return false;
    }
};